### PR TITLE
Views and documents for couch db,

### PR DESCRIPTION
**Pull request by georgethomas111**
Have added some comments. to README. 

Update View needs to be worked on.

### CODE REVIEW

**Comment by georgethomas111** [comment]
@johnhof

File: README.md [hunk 1]
```diff
@@ -1,2 +1,22 @@
 # couchdb
 Minimalist CouchDB client
+
+## Understanding.
+
+client.go --> represents the couchdb client for the others to use.
```
Comment (johnhof) [inline README.md]: Might want to make this into a bulleted list. Github just smashes consecutive lines into  a paragraph (i know, its silly)

```
- client.go - represents the couchdb client for the others to use.
- db.go - This is the low level db interface for the object. (Couch works on rest.)
- dbrequests - Wrapper for all requests from couch.
- view.go - Deals with creation of views.
- document.go - Deals with the creation and updating of documents in the db.
```

becomes
- client.go - represents the couchdb client for the others to use.
- db.go - This is the low level db interface for the object. (Couch works on rest.)
- dbrequests - Wrapper for all requests from couch.
- view.go - Deals with creation of views.
- document.go - Deals with the creation and updating of documents in the db.

Comment (johnhof) [inline README.md]: maybe wrap the files in `**FILE_NAME**` as well to get
- **client.go** - represents the couchdb client for the others to use.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,22 @@
...
+view.go --> Deals with creation of views.
+document.go --> Deals with the creation and updating of documents in the db.
+
+## Running Tests. 
+
+Make sure there is a couch instane running on default port (5984) for it to work. 
+
+Have disabled the DeleteDB for now so the DB created would persist.
+
+$ export GOPATH=$PWD
```
Comment (johnhof) [inline README.md:17]: Same thing here

``````
\```bash
$ export GOPATH=$PWD
$ go get -a
$ go test -v
\```
``````

becomes

``` bash
$ export GOPATH=$PWD
$ go get -a
$ go test -v
```

**Comment by johnhof** [comment]
My only other suggestion is to add some basic usage examples. Overall, :100:

**Pull request merged**